app/pages/exportImport/PerformImport.tsx

```diff
@@ -228,15 +253,24 @@ export default function PerformImport({ location }: Props) {
                         </Columns.Column>
                         <Columns.Column>
                             <div className={styles.importedList}>
-                                <div className="flexChildFill flexColumn justifyCenter">
+                                <div className={styles.importedListInner}>
                                     {importedData.identities.map(
                                         (identity: Identity) => (
                                             <div
                                                 key={identity.id}
                                                 className={styles.importSection}
                                             >
-                                                <h3>
-                                                    <b>ID:</b> {identity.name}
+                                                <h3 className="mB0">
+                                                    <b>ID:</b> {identity.name}{' '}
+                                                    {messages[identity.id] && (
+                                                        <span className="bodyLight textFaded body3">
```

Comment:
You probably meant to do this but forgot, I assume

```suggestion
                                                        <span className="bodyLight textFaded">
```